mods/cnc/fluent/rules.ftl

```diff
@@ -782,71 +782,134 @@ bot-hal9001 =
-label-cnc-map-generator-choice-terrain-type-overgrown = Overgrown
-label-cnc-map-generator-choice-terrain-type-rocky = Rocky
-label-cnc-map-generator-choice-terrain-type-mountains = Mountains
-label-cnc-map-generator-choice-terrain-type-mountain-lakes = Mountain Lakes
+label-cnc-map-generator-choice-terrain-type-lakes =
+   .label = Lakes
+   .description = Open spaces with moderately sized lakes
+label-cnc-map-generator-choice-terrain-type-puddles =
+   .label = Puddles
+   .description = Open spaces with small ponds
+label-cnc-map-generator-choice-terrain-type-gardens =
+   .label = Gardens
+   .description = Featureful terrain with ponds, cliffs, and forests
+label-cnc-map-generator-choice-terrain-type-plains =
+   .label = Plains
+   .description = Open spaces with few trees and occasional cliffs
+label-cnc-map-generator-choice-terrain-type-parks =
+   .label = Parks
+   .description = Open spaces with light forestry and occasional cliffs
+label-cnc-map-generator-choice-terrain-type-woodlands =
+   .label = Woodlands
+   .description = Moderate forestry with occasional cliffs
+label-cnc-map-generator-choice-terrain-type-overgrown =
+   .label = Overgrown
+   .description = Dense forestry with narrow passages and moderate cliffs
+label-cnc-map-generator-choice-terrain-type-rocky =
+   .label = Rocky
+   .description = Moderate cliffs with light forestry
+label-cnc-map-generator-choice-terrain-type-mountains =
+   .label = Mountains
+   .description = Long and densely packed cliffs
+label-cnc-map-generator-choice-terrain-type-mountain-lakes =
+   .label = Mountain Lakes
+   .description = Long and densely packed cliffs surrounding lakes
 
 label-cnc-map-generator-option-rotations = Rotations
 
 label-cnc-map-generator-option-mirror = Mirror
-label-cnc-map-generator-choice-mirror-none = None
-label-cnc-map-generator-choice-mirror-left-matches-right = Left vs right
-label-cnc-map-generator-choice-mirror-top-left-matches-bottom-right = Top left vs bottom right
-label-cnc-map-generator-choice-mirror-top-matches-bottom = Top vs bottom
-label-cnc-map-generator-choice-mirror-top-right-matches-bottom-left = Top right vs bottom left
+label-cnc-map-generator-choice-mirror-none =
+   .label = None
+label-cnc-map-generator-choice-mirror-left-matches-right =
+   .label = Left vs right
+label-cnc-map-generator-choice-mirror-top-left-matches-bottom-right =
+   .label = Top left vs bottom right
+label-cnc-map-generator-choice-mirror-top-matches-bottom =
+   .label = Top vs bottom
+label-cnc-map-generator-choice-mirror-top-right-matches-bottom-left =
+   .label = Top right vs bottom left
 
 label-cnc-map-generator-option-shape = Bounds Shape
-label-cnc-map-generator-choice-shape-square = Square
-label-cnc-map-generator-choice-shape-circle-mountain = Circle in mountains
-label-cnc-map-generator-choice-shape-circle-water = Circle in water
+label-cnc-map-generator-choice-shape-square =
+   .label = Rectangle
+   .description = Full rectangular terrain generation
```

Comment:
```suggestion
   .description = Full terrain generation
```

It's in the name

Reply:
I've changed it to "Full _size_ terrain generation".